Automattic/jetpack
# VideoPress: fix visual issue in the VideoPress icons

This PR fixes a visual issue in the VideoPress icon when rendered in the MediaPlacehoder. Introduced here https://github.com/Automattic/jetpack/pull/27687


#### Changes proposed in this Pull Request:

* VideoPress: fix visual issue in the VideoPress icons

#### Other information:

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the block editor
* Create a new block
* Confirm the VP logo looks as expcted

before | after
-----------|-------
<img width="644" alt="Screen Shot 2022-12-10 at 11 15 02" src="https://user-images.githubusercontent.com/77539/206852234-392c07e9-470b-461c-ad2c-2e2a643b9171.png">| <img width="649" alt="image" src="https://user-images.githubusercontent.com/77539/206852215-d784ac82-164e-47f9-8a09-26925f3e167a.png">
